Tasks/Common/securefiles-common/bump-task-version.ps1

```diff
@@ -0,0 +1,37 @@
+# Bumps the version of all of the tasks that are technically modified when securefiles-common is changed.
+# Please validate the updated versions to confirm that nothing went horribly wrong.
+Param(
+[parameter(Mandatory=$true)]
+# The current sprint, find at https://whatsprintis.it/
+[int] $currentSprint,
+# The path to the Tasks folder
+[parameter()]
+[string] $taskRoot = (Join-Path $PSScriptRoot "..\..")
+)
+
+# Bump patch version and adjust sprint
+"AndroidSigningV2", "AndroidSigningV3", "DownloadSecureFileV1", "HelmDeployV0", "InstallAppleCertificateV2", "InstallAppleProvisioningProfileV1", "InstallSSHKeyV0" | % {
+    $taskLocation = Join-Path "$taskRoot/$_" "task.json"
+    $taskContent = Get-Content $taskLocation 
+    $task = $taskContent | ConvertFrom-Json
+    $v = $task.version
+    $newPatch = 0
+    if ($v.Minor -eq $currentSprint) {
+        $newPatch = $v.Patch + 1
+    }
+    Write-Output "${_}: $($v.Major).$($v.Minor).$($v.Patch) to $($v.Major).$currentSprint.$newPatch"
+    # Doing an awkward string replace to not interfere with any custom task.json formatting
+    $versionMatch = $taskContent | Select-String '"version":'
+    $versionLine = $versionMatch.LineNumber
+    $taskContent[$versionLine+1] = $taskContent[$versionLine+1].Replace(" $($v.Minor),", " $currentSprint,")
+    $taskContent[$versionLine+2] = $taskContent[$versionLine+2].Replace(" $($v.Patch)", " $newPatch")
+    $taskContent | Set-Content $taskLocation
+    
+    $taskLocLocation = Join-Path "$taskRoot/$_" "task.loc.json"
+    $taskLocContent = Get-Content $taskLocLocation 
+    $versionMatch = $taskLocContent | Select-String '"version":'
+    $versionLine = $versionMatch.LineNumber
+    $taskLocContent[$versionLine+1] = $taskLocContent[$versionLine+1].Replace(" $($v.Minor),", " $currentSprint,")
```

Comment:
There are cases where the `.loc` file was not updated, and this will may miss those.  It would be nice to print or error out if the match isn't found in the `.loc` file.

Reply:
Cases in this PR or just in general?

As far as the script goes, I stole it from packaging-common. I'll see if I can fix it up to print some nice errors for the user.

Reply:
In general.  I've seen cases (perhaps caused some myself) where the task.json's version was changed, but the corresponding task.loc.json was not updated.  This does not cover that case AFAICT, and the `.loc` won't get updated.

Reply:
If the `.loc` file doesn't exist (or the normal `task.json` file), then this will actually fail at the `Get-Content` step with an error like: `Get-Content : Cannot find path task.loc.json' because it does not exist.`

If the `.loc` file doesn't have a version string, then this will fail silently but I think that's a bigger issue and not the responsibility of this script (It's really just meant to help with some of the tedious parts of updating common tasks). The script does also mention that all the results of should be verified